src/pages/Languages/index.tsx

```diff
@@ -109,8 +106,16 @@ export const Languages = () => {
         })}
       </LangContainer>
       <p>
-        If you'd like to see the launchpad in another language, or if you can
-        help translate, get in touch.
+        <FormattedMessage
```

Comment:
@ryancreatescopy @wackerow - we can use HTML elements & React components within translation phrases! 🤯 🎉

Reply:
Hm... so will the translator be able to see the whole context, and also be able to translate that variable (ie: `getInTouch`) so it stays a variable?
If so looks like this would work great, and could also be used to keep inline styling. =)